app/models/manageiq/providers/ansible_tower/shared/automation_manager/openstack_credential.rb

```diff
@@ -0,0 +1,49 @@
+module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::OpenstackCredential
+  extend ActiveSupport::Concern
```

Comment:
I see you are following the pattern in the other credential modules, but why is `AS::Concern` used here?